borg/remote.py

```diff
@@ -191,6 +191,10 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self.rollback()
         self.close()
 
+    @property
+    def id_str(self):
+        return bin_to_hex(self.id)
```

Comment:
__str__ is used at some places for a much more verbose representation that is used in screen output. archive and stats come to mind.